stock_by_warehouse_stock/18n/es.po

```diff
@@ -0,0 +1,61 @@
+# Translation of Odoo Server.
```

Comment:
Wrong file name, directory should be `i18n`

Reply:
Done :see_no_evil: